integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/spring/cloud/SpringCloudProtocolResolver.java

```diff
@@ -0,0 +1,71 @@
+package no.difi.meldingsutveksling.serviceregistry.spring.cloud;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This file "SpringCloudProtocolResolver.java" should be located in "no/difi/meldingsutveksling/serviceregistry/spring/cloud" directory, not in "/var/lib/jenkins/jobs/move-ci-integrasjonspunkt/workspace/repo/integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/spring/cloud". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1598)